src/Microsoft.AspNetCore.Razor.Language/CodeGeneration/RuntimeTagHelperWriter.cs

```diff
@@ -445,8 +445,6 @@ public override void WriteSetTagHelperProperty(CSharpRenderingContext context, S
             }
         }
 
-        private static string GetTagHelperVariableName(string tagHelperTypeName) => "__" + tagHelperTypeName.Replace('.', '_');
-
         protected static string GetTagHelperPropertyAccessor(
```

Comment:
Still need to be protected? Just asking because I saw you nuke the privateness of the design time variant.